tests/cloud/conftest.py

```diff
@@ -58,15 +59,17 @@ def __init__(self,model_name,num_cores,prompt,prompts_txt_file_path,aic_enable_d
         self.batch_size = batch_size
         self.prompt_len = prompt_len
         self.ctx_len = ctx_len
+        self.generation_len = None
         self.mxfp6 = mxfp6
         self.mxint8 = mxint8
+        self.full_batch_size = None
         self.device_group = device_group
 
     def model_card_dir(self):
         return str(os.path.join(QEFF_MODELS_DIR, str(self.model_name)))
     
     def qpc_base_dir_name(self):
-        return get_qpc_dir_name_infer(self.num_cores, self.mos, self.batch_size, self.prompt_len, self.ctx_len, self.mxfp6, self.mxint8, self.device_group)
+        return get_qpc_dir_name_infer(self.num_cores, self.mos, self.batch_size, self.prompt_len, self.ctx_len, self.mxfp6, self.mxint8, self.device_group, self.full_batch_size)

```

Comment:
is pre-commit linter run on it? looks like code is not formatted. pls run the pre commit.

Reply:
It is done.